### PR TITLE
New version: Ablaze.Floorp.Daylight version 11.8.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135131)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/a/Ablaze/Floorp/Daylight/11.8.0/Ablaze.Floorp.Daylight.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- daylight
+- firefox-fork
+- floorp
+- foss
+- gecko
+- internet-browser
+- open-source
+- web
+- web-browser
+ReleaseNotes: '"Floorp Daylight": v11.8.0 (Firefox v115.7.0)This is NOT the release (stable) channel version of Floorp. This is a beta build of Floorp. It is intended for testing purposes only. It is not recommended that you use this version of Floorp for daily browsing. If you would like to download the release version of Floorp, please visit floorp.app.'
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/Daylight/11.8.0/Ablaze.Floorp.Daylight.locale.en-US.yaml:31]:
```suggestion
```

File: manifests/a/Ablaze/Floorp/Daylight/11.8.0/Ablaze.Floorp.Daylight.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ReleaseNotesUrl: https://github.com/Floorp-Projects/Floorp/releases/tag/beta
```
Comment (Hibi-10000) [inline manifests/a/Ablaze/Floorp/Daylight/11.8.0/Ablaze.Floorp.Daylight.locale.en-US.yaml:32]:
```suggestion
```

**Review by Hibi-10000 — CHANGES_REQUESTED** [review]
No release notes for Floorp Daylight.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-18T05:30:22.0840108Z ##[error] Manifest Error: The multi file manifest has inconsistent field values. [PackageVersion] Value: 11.7.1 File: Ablaze.Floorp.Daylight.locale.ja-JP.yaml

(Automated response - build 602.)

File: manifests/a/Ablaze/Floorp/Daylight/11.8.0/Ablaze.Floorp.Daylight.locale.ja-JP.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Created with komac v2.0.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.5.0.schema.json
+
+PackageIdentifier: Ablaze.Floorp.Daylight
+PackageVersion: 11.7.1
```
Comment (stephengillie) [inline manifests/a/Ablaze/Floorp/Daylight/11.8.0/Ablaze.Floorp.Daylight.locale.ja-JP.yaml:5]:
```suggestion
PackageVersion: 11.8.0
```

**Pull request closed without merging**